common/azure-core-test-perf/src/main/java/com/azure/core/test/perf/PerfStressOptions.java

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.test.perf;
+
+import com.beust.jcommander.Parameter;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+/**
+ * Represents the command line configurable options for a performance test.
+ */
+@JsonPropertyOrder(alphabetic = true)
+public class PerfStressOptions {
+    @Parameter(names = { "-d", "--duration" }, description = "duration of test in seconds")
+    private int duration = 10;
+
+    @Parameter(names = { "--host" }, description = "Host to redirect HTTP requests")
+    private String host;
+
+    @Parameter(names = { "--insecure" }, description = "Allow untrusted SSL server certs")
+    private boolean insecure = false;
+
+    @Parameter(names = { "-i", "--iterations" }, description = "Number of iterations of main test loop")
+    private int iterations = 1;
+
+    @Parameter(names = { "--no-cleanup" }, description = "Disables test cleanup")
+    private boolean noCleanup = false;
+
```

Comment:
shouldn't cleanup be turned on by default ?  i.e. `noCleanup = true`

Reply:
it is turned on by default.
This flag is to indicate if you don't want the cleanup.